PRASCore.jl/src/Systems/assets.jl

```diff
@@ -1,6 +1,38 @@
 abstract type AbstractAssets{N,L,T<:Period,P<:PowerUnit} end
 Base.length(a::AbstractAssets) = length(a.names)
 
+function Base.show(io::IO, asset_collection::T) where {T<:AbstractAssets}
```

Comment:
Can this just be defined on super type AbstractAssets, why do we need the where .....?